src/Word/Service/SeparatorToSeparatorFactory.php

```diff
@@ -23,7 +24,7 @@ public function __construct($creationOptions = null)
             return;
         }
 
-        if ($creationOptions instanceof Traversable) {
+        if ($creationOptions instanceof \Traversable) {
```

Comment:
You're correct, I was deciding between either approach and forgot to clean up `\Traversable` when I realized importing was preferred.